### DISCUSSION TITLE
Bug in editing ents pressing numbers on french keyboard

### DISCUSSION THREAD

**Discussion by FabioOG** [redeclipse/base#1148] [Help and Support] [answered]
This is quite embarrassing in dev version, because i the only ways to edit ents are /entset and press ing a number and scroll, and the number way is the only good way to find a mapmodel type for example.
I thought understand that on US keyboards there are 1, 2, 3 keys that are used to edit ents in RE, but on french keyboards to get 1 we use in fact shift+&.
This changes nothing for most of case because we have /entset but it's easier to find what we want for the ent by scrolling possibilities, instead of make tries with /entset.

**Comment by qreeves** [comment, marked as answer]
Probably the best way to tackle this is to allow those buttons to be remapped like other keys.

That being said, we will likely have an on-screen interface for this in the future using the new UI system.

**Comment by FabioOG** [comment]
Ty, i got the same problem with the cmd that appears using / button that dont exist on french keyboards and i have been able to solve it by the controls interface
